motoko/cert-var/README.md

```diff
@@ -0,0 +1,54 @@
+# Certified Variable
+
+![Compatibility](https://img.shields.io/badge/compatibility-0.7.0-blue)
+[![Build Status](https://github.com/dfinity/examples/workflows/motoko-cert-var-example/badge.svg)](https://github.com/dfinity/examples/actions?query=workflow%3Amotoko-cert-var-example)
+
+The example shows a cryptographically certified variable.
```

Comment:
```suggestion
The example demonstrates the use of a single cryptographically certified variable, as supported by the Internet Computer. 
```
And maybe ad links to the IC Public Spec section(s) about them (either here or below) and perhaps the video explaining their use.